extensions/fhir/deployment/src/main/java/org/apache/camel/quarkus/component/fhir/deployment/FhirProcessor.java

```diff
@@ -26,15 +26,27 @@
 import io.quarkus.deployment.builditem.FeatureBuildItem;
 import io.quarkus.deployment.builditem.nativeimage.NativeImageResourceBundleBuildItem;
 import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.RuntimeInitializedClassBuildItem;
 
 final class FhirProcessor {
     private static final String FEATURE = "camel-fhir";
 
+    private static final String[] RUNTIME_INITIALIZED_CLASSES = new String[] {
+            "org.apache.http.impl.auth.NTLMEngineImpl"
```

Comment:
This is already handled [elsewhere](https://github.com/apache/camel-quarkus/blob/43960d8e5bafbfd8c42a33ee08fdf79a033a831e/extensions-support/httpclient/deployment/src/main/java/org/apache/camel/quarkus/support/httpclient/deployment/HttpClientProcessor.java). So it's better to make the fhir extension depend on `camel-quarkus-support-httpclient`.

Reply:
Thanks @jamesnetherton !

I will revise accordingly